src/components/Profile.jsx

```diff
@@ -0,0 +1,54 @@
+import Alert from 'react-bootstrap/Alert';
+import { useSelector } from 'react-redux';
+import ListGroup from 'react-bootstrap/ListGroup';
+import style from '../assets/style/Profile.module.css';
+
+function Profile() {
+  const { rocketsArray } = useSelector((state) => state.rockets);
+
+  const { missions } = useSelector((state) => state.missions);
+
```

Comment:
- [ OPTIONAL ] Great job so far @Graciano1997  and @mumbereh   Your project looks great :clap: There is a minor change I will suggest you implement that was missed by the previous reviewers :wink: 
- [ ] Per the project requirements, 

> Actions should be dispatched once
> Data should not be added to the store on every re-render (i.e. when changing views / using navigation).
> Rockets is the default view, so its data would be fetched when the application starts. However, the mission data should only be fetched (once) when a user navigates to the Missions section.

- [ ] However, I noticed that on both your Missions page and Rockets page, when missions are joined or Rockets are reserved and I change views, the joined missions and reserved rockets all change back to their default values. 
- [ ] However, per the project requirements, both the `My Missions ` and `My Rockets` shouldn't change but stay the same regardless of the switching between views and as a result meet the project requirements. :+1: 
- [ ] This means reserved rockets and joined missions will still be joined or reserved and still show on the `My Profile` page even if a user switches between the pages :+1: 
- [ ] From the video below, you can see that even though I reserved rockets and joined missions, I was told I hadn't join any once I switched back to the rockets page :+1: 
- Video Reference

[Screencast from 09-09-2023 04:49:44 PM.webm](https://github.com/Graciano1997/space_travelers_hub/assets/102699958/fc0685a0-5398-45a9-95d7-d158373bdb04)